lib/cosmos/tools/handbook_creator/handbook_creator.rb

```diff
@@ -36,6 +35,8 @@ def initialize (options)
         System.commands
         @config = HandbookCreatorConfig.new(options.config_file)
         ConfigParser.splash = nil
+        # Copy the assets to the local outputs directory
+        FileUtils.cp_r(File.join(::Cosmos::USERPATH,'config','tools','handbook_creator','assets'), System.paths['HANDBOOKS'])
```

Comment:
So how about just copying everything over when we launch?

Reply:
This would is fine, except it needs to be only if the files are not already present (and should be at output generation time, not tool initialization time).   I don't want to overwrite any intentional changes, which is why the existing files should be checked for existence first.